packages/button/index.js

```diff
@@ -34,10 +35,18 @@ class WarpButton extends kebabCaseAttributes(WarpElement) {
     rel: { type: String, reflect: true },
     fullWidth: { type: Boolean, reflect: true },
     buttonClass: { type: String, reflect: true },
+    name: { type: String, reflect: true },
+    value: { type: String, reflect: true },
   };
 
   static styles = [WarpElement.styles];
 
+  updated(changedProperties) {
+    if (changedProperties.has('value')) {
+      this.setValue(this.value);
+    }
+  }
```

Comment:
any time value is set or changed, either via the attribute or the property, we need to update the form internals using the setValue method provided by the open wc mixin. so Long as we do this, the form will capture name and value for us if set.